protocols/relay/examples/Dockerfile

```
@@ -0,0 +1,22 @@
+# 1: Build the exe
+FROM rust:latest as builder
```

```suggestion
FROM rust:1 as builder
```

To make sure that docker does not upgrade to a breaking change automatically.

---

Ok , but why you choose version 1 and not the current available latest version? (1.52.1)

---

The Rust project follows [semantic versioning](https://semver.org/). In this particular case this means that e.g. an image with `v1.52.1` does not introduce any breaking changes compared to an image with `v1.52.0` or `v1.51.0`. As far as I am aware the `1` tag is overwritten each time a new image in the `v1.x.x` series is released. Thus pointing to `rust:1` instead of `rust:latest` will always give us the latest image from the `v1.x.x` series, but will not automatically update to the `v2.x.x` series given that the `v2.x.x` series might include breaking changes.

@r-zig does that make sense?